app/celer-export-geant.cc

```diff
@@ -56,8 +56,13 @@ int main(int argc, char* argv[])
         GeantImporter import(GeantSetup(gdml_input_filename, options));
         RootExporter  export_root(root_output_filename.c_str());
 
+        GeantImporter::DataSelection selection;
+        selection.particles   = GeantImporter::DataSelection::em;
+        selection.processes   = GeantImporter::DataSelection::em;
+        selection.reader_data = false;
```

Comment:
I don't think we want this to be the default for the geant exporter: it's more of a special case for generating "lite" data for unit tests.